eng/common/TestResources/New-TestResources.ps1

```diff
@@ -83,7 +83,14 @@ param (
     [switch] $OutFile,
 
     [Parameter()]
-    [switch] $SuppressVsoCommands = ($null -eq $env:SYSTEM_TEAMPROJECTID)
+    [switch] $SuppressVsoCommands = ($null -eq $env:SYSTEM_TEAMPROJECTID),
+
+    # Captures any arguments not declared here (no parameter errors)
+    # This enables backwards compatibility with old script versions in
+    # hotfix branches if and when the dynamic subscription configuration
+    # secrets get updated to add new parameters.
+    [Parameter(ValueFromRemainingArguments = $true)]
+    $NewTestResourcesRemainingArguments
```

Comment:
The name seems unnecessarily descriptive. Any problem with just `$RemainingArguments` or `$UnrecognizedArguments`?

Reply:
Nope, just copied it from `RemoveTestResourcesRemainingArguments`. I can change that one too as well.

Reply:
Actually, is there any chance the name is specific to avoid overlaps with test-resources-post scripts that use `$RemainingArguments`?